src/Datastore/DatastoreSessionHandler.php

```diff
@@ -0,0 +1,299 @@
+<?php
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Datastore;
+
+use Exception;
+use InvalidArgumentException;
+use SessionHandlerInterface;
+
+/**
+ * Custom session handler backed by Cloud Datastore.
+ *
+ * {@see http://php.net/manual/en/class.sessionhandlerinterface.php}
```

Comment:
This tag doesn't parse properly. It should go at the bottom of this block (after the examples), and be formatted as follows:

```
* @see http://php.net/manual/en/class.sessionhandlerinterface.php SessionHandlerInterface
```

This will cause it to render like this:

![screen shot 2016-10-31 at 4 18 07 pm](https://cloud.githubusercontent.com/assets/89034/19870133/a3f8e462-9f85-11e6-9813-8a61302155e8.png)

Sorry I missed this earlier. I just ran the doc parser and took a look at the output.